lib/wsdl.js

```diff
@@ -1130,7 +1130,7 @@ WSDL.prototype._processNextInclude = function(includes, callback) {
     includePath = url.resolve(self.uri, include.location);
   }
 
-  open_wsdl(includePath, this.options, function(err, wsdl) {
```

Comment:
open_wsdl is used in lib/soap, tests started to fail as I modified the original function, thoughts on current changes?